src/Symfony/Component/Cache/Traits/RedisTrait.php

```diff
@@ -246,6 +251,12 @@ public static function createConnection($dsn, array $options = [])
         } elseif (is_a($class, \Predis\Client::class, true)) {
             if ($params['redis_cluster']) {
                 $params['cluster'] = 'redis';
+                if (isset($params['redis_sentinel'])) {
+                    throw new InvalidArgumentException(sprintf('Invalid Redis DSN: cannot use both redis_cluster and redis_sentinel at the same time: %s', $dsn));
```

Comment:
Since redis_sentinel/cluster can also be provided as options, the message should be a bit less specific I think, e.g.:
`Cannot use both "redis_cluster" and "redis_sentinel" at the same time: %s`